app/src/main/java/in/testpress/testpress/ui/adapters/DashboardAdapter.java

```diff
@@ -57,10 +58,11 @@ public int getItemViewType(int position) {
                 return OFFERS_CAROUSEL;
             case "chapter_content_attempt":
                 return CONTENT_CAROUSEL;
-            case "trophy_leaderboard":
-                return LEADERBOARD_LIST;
             case "products":
                 return COURSE_CAROUSEL;
+            case "trophy_leaderboard":
+                if (instituteSettings.getLeaderboardEnabled())
+                return LEADERBOARD_LIST;
```

Comment:
Where is the logic to hide leaderboard? Did you check whether this hides leaderboard?

Reply:
Yes, this will hide the leaderboard since we are not returning when leaderboard enabled is false